Spigot-API-Patches/0196-Player-elytra-boost-API.patch

```diff
@@ -0,0 +1,35 @@
+From 1b90910d523a5e8ff429c7b0dde0cde8ec3c4a6f Mon Sep 17 00:00:00 2001
+From: Trigary <trigary0@gmail.com>
+Date: Tue, 14 Apr 2020 12:06:14 +0200
+Subject: [PATCH] Player elytra boost API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 3600b4c8..bde3ca97 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1635,6 +1635,21 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     public void openBook(@NotNull ItemStack book);
+ 
++    // Paper start
++    /**
++     * Boost a Player that's {@link #isGliding()} using a {@link Firework}.
++     * If the creation of the entity is cancelled, no boosting is done.
++     * This method does not fire {@link com.destroystokyo.paper.event.player.PlayerElytraBoostEvent}.
++     *
++     * @param firework The {@link Material#FIREWORK_ROCKET} to boost the player with
++     * @return The {@link Firework} boosting the Player or null if the spawning of the entity was cancelled
++     * @throws IllegalArgumentException if {@link #isGliding()} is false
++     * or if the {@code firework} isn't a {@link Material#FIREWORK_ROCKET}
++     */
++    @Nullable
++    Firework boostElytra(@NotNull ItemStack firework);
++    // Paper end
```

Comment:
You probably don't need to end the Paper comment section and just remove the following paper start.

Reply:
moved the method to the bottom of the paper additions